### PR TITLE
Disassemble the BLUEMONS.GB debug ROM

### LINKED ISSUE #272: Disassemble the final English debug ROMs
The Gen 1 leak included two debug ROM builds:

- BLUEMONS.GB `sha1: 5B1456177671B79B263C614EA0E7CC9AC542E9C4`
- YELLMONS.GBC `sha1: D44E96EDDFBDAD633CBE4E6E64915E9E198974B0`

These could be disassembled and added as build options to pokered and pokeyellow, without ever having to refer to original source code.

This is also possible for pokegold (https://github.com/pret/pokegold/pull/56) and pokecrystal (https://github.com/pret/pokecrystal/pull/743).

### PR DESCRIPTION
**Pull request by Rangi42**
Resolves #272.

Press Select at the title screen to open the debug menu.

Only BLUEMONS.GB was available for comparison, so this does not build a debug Red ROM.

Some changes to free space:
- Sections are padded with $FF, not $00.
- However, the `rst` and interrupt sections are still padded with $00.
- 32 zero bytes (two blocks) at the end of plateau.bst were actually free space.

### CODE REVIEW

File: home/npc_movement.asm [hunk 1]
```diff
@@ -50,5 +50,15 @@ RunNPCMovementScript::
 EndNPCMovementScript::
 	farjp _EndNPCMovementScript
 
-EmptyFunc2::
+DebugPressedOrHeldB::
+IF DEF(_DEBUG)
+	ld a, [wd732]
+	bit 1, a
```
Comment (mid-kid) [inline home/npc_movement.asm:56]: This is obviously the Field Debug flag. See `wDebugFlags` in pokecrystal.

Comment (Rangi42) [reply home/npc_movement.asm:56]: It's used for other things though, like whether play time is being counted, whether the Bicycle is force-enabled, what type of warp is going on, etc.

Comment (mid-kid) [reply home/npc_movement.asm:56]: This specific bit is?

Comment (Rangi42) [reply home/npc_movement.asm:56]: The byte is. It's one of many instances in pokered where named constants don't exist yet. I'd rather not create one just for that specific bit, but also don't want this to become a general improvement PR that identifies the other bits.

**Pull request merged**